pkg/plugins/manifest.go

```diff
@@ -82,49 +84,130 @@ func readPluginManifest(body []byte) (*pluginManifest, error) {
 	return manifest, nil
 }
 
+func getChksums(dpath, manifestPath string, log log.Logger) (map[string]string, error) {
```

Comment:
while not exported, I think the complexity of this warrants a func comment explaining what it does.

Reply:
Thanks @kylebrandt, will fix it.